pandas/tests/indexing/test_chaining_and_caching.py

```diff
@@ -261,20 +289,32 @@ def random_text(nobs=100):
         df["letters"] = df["letters"].apply(str.lower)
         assert df._is_copy is None
 
+    @pytest.mark.arm_slow
+    def test_detect_chained_assignment_str(self):
+
         df = random_text(100000)
         indexer = df.letters.apply(lambda x: len(x) > 10)
         df.loc[indexer, "letters"] = df.loc[indexer, "letters"].apply(str.lower)
 
+    @pytest.mark.arm_slow
+    def test_detect_chained_assignment_is_copy(self):
+
         # an identical take, so no copy
         df = DataFrame({"a": [1]}).dropna()
         assert df._is_copy is None
         df["a"] += 1
 
+    @pytest.mark.arm_slow
```

Comment:
might be worthile to move the slow to a separate file and just mark it

Reply:
im not wild about that pattern, but dont have a better alternative ATM.  im happy enough with how it is here

Reply:
i c. maybe leave these decorators, but split anyhow just to make it more obvious?

Reply:
i think the follow-up is to see if there's only a subset of these that are actually slow